src/landscape/components/LandscapeProfile/ProfileCard.js

```diff
@@ -0,0 +1,136 @@
+import React, { useMemo } from 'react';
+
+import _ from 'lodash/fp';
+import { useTranslation } from 'react-i18next';
+import { Link as RouterLink } from 'react-router-dom';
+import { getTermLabel } from 'taxonomies/taxonomiesUtils';
+
+import { Button, Card, CardContent, Grid, Typography } from '@mui/material';
+
+import { countryNameForCode } from 'common/utils';
+import Restricted from 'permissions/components/Restricted';
+
+const FIELDS = [
+  {
+    label: 'landscape.profile_profile_card_location_label',
+    customValue: landscape => countryNameForCode(landscape.location)?.name,
+  },
+  {
+    label: 'landscape.profile_profile_card_area_types_label',
+    customValue: (landscape, { t }) =>
+      _.isEmpty(landscape.areaTypes)
+        ? null
+        : landscape.areaTypes
+            .map(areaType =>
+              t(`landscape.profile_profile_card_area_types_${areaType}`)
+            )
+            .join(', '),
+  },
+  {
+    label: 'landscape.profile_profile_card_ecosystem_types_label',
+    customValue: (landscape, { i18n }) =>
+      getTermsList('ecosystem-type', landscape, i18n),
+  },
+  {
+    label: 'landscape.profile_profile_card_languages_label',
+    customValue: (landscape, { i18n }) =>
+      getTermsList('language', landscape, i18n),
+  },
+  {
+    path: 'population',
+    label: 'landscape.profile_profile_card_population_label',
+  },
+  {
+    label: 'landscape.profile_profile_card_livelihoods_label',
+    customValue: (landscape, { i18n }) =>
+      getTermsList('livelihood', landscape, i18n),
+  },
+  {
+    label: 'landscape.profile_profile_card_commodities_label',
+    customValue: (landscape, { i18n }) =>
+      getTermsList('commodity', landscape, i18n),
+  },
+];
+
+const getTermsList = (type, landscape, i18n) =>
+  _.isEmpty(landscape?.taxonomyTerms[type])
+    ? null
+    : landscape?.taxonomyTerms[type]
+        .map(term => getTermLabel(term, i18n.resolvedLanguage))
+        .join(', ');
+
+const ProfileField = props => {
+  const translation = useTranslation();
+  const { path, label, landscape, customValue } = props;
+
+  const value = useMemo(
+    () => (customValue ? customValue(landscape, translation) : landscape[path]),
+    [customValue, landscape, path, translation]
+  );
+
+  if (!value) {
+    return null;
+  }
+
+  return (
+    <>
+      <Grid
+        item
+        sx={{ textTransform: 'uppercase' }}
+        xs={5}
+        component={Typography}
+        variant="caption"
+      >
+        {label}
+      </Grid>
+      <Grid item xs={7}>
+        {value}
+      </Grid>
+    </>
+  );
+};
+
+const ProfileCard = props => {
+  const { t } = useTranslation();
+  const { landscape } = props;
+
+  return (
+    <Card
+      component="section"
+      aria-label={t('landscape.profile_profile_card_label')}
+      style={{
+        width: '100%',
```

Comment:
question: is 100% the default width?

Reply:
@paulschreiber 100% meaning that it will use all the space available within its grid item, but yes, that is the default right now? Or are you asking if it will behave the same if we remove this?

Reply:
Yes — asking if the behaviour is the same if you remove that line.

Reply:
Let me check this